.github/workflows/linux.yml

```diff
@@ -1,6 +1,25 @@
 name: Linux builds
 
-on: [push, pull_request]
+on:
+  push:
+    paths-ignore:
+      - '.clang-format'
+      - '.mdl-styles'
+      - '*.md'
+      - 'docs/**'
+      - 'licenses/**'
+      - 'website/**'
+      - 'scripts/**'
```

Comment:
Are there some scripts here that get run in the workflow? Such as `scripts/create-package.sh` for example. It might be good to not ignore them if that's the case.

(same comment for the other workflows)

Reply:
Good point. I'll double-check that and/or remove the whole `scripts/**` entry perhaps because then maintaining per-filename exceptions per workflow could become quite fragile and error-prone.

But if it's just a few scripts, maybe I'll keep it anyway. I'll see.

Reply:
I've just removed `scripts/**` from the ignore list. There's not direct support for ignoring the contents of a directory _except_ a few script files, and even if it was, maintaining that exclusion list is too error prone.

The scripts don't change too often, so I think this is fine. Not rebuilding on doco changes (which are much more frequent) is the main improvement here.